.github/workflows/integration.yml

```diff
@@ -0,0 +1,28 @@
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+  pull_request:
+  merge_group:
+    types: [checks_requested]
+permissions: read-all
+name: Integration
+jobs:
+  test-arch-linux:
+    name: Run on Arch Linux
+    runs-on: ubuntu-latest
+    container:
+      image: archlinux:latest
+    steps:
+      - uses: actions/checkout@v6
+      - name: Install dependencies
+        run: pacman -Syu --noconfirm base-devel rustup
+      - name: Install Rust toolchain
+        run: rustup default stable
+      - name: Build rv
+        run: cargo build --release
```

Comment:
I recommend building with `--locked` as well. This ensures that your dependencies are [reproducible](https://wiki.archlinux.org/title/Reproducible_builds) and match the exact versions recorded in the `Cargo.lock` file.